tools/opt/CMakeLists.txt

```diff
@@ -7,6 +7,8 @@ set(LLVM_LINK_COMPONENTS
   DXIL # HLSL Change
   DxcBindingTable # HLSL Change
   HLSL # HLSL Change
+  DxilContainer #  HLSL Change for DxcOptimizerPass
+  DxilRootSignature #  HLSL Change for DxcOptimizerPass
```

Comment:
I didn't look into exactly which passes require these, but I take your word for it 😄

Reply:
I'm not sure which pass too. But I'm sure without these two libs the linking will fail.